### PR TITLE
Convert (known) file hash playbook

### PR DESCRIPTION
**Pull request by David-BMS**
## Status
In Progress/In Hold

## Related Issues
#19360

## Description
The playbook accepts a hash or array of hashes and provides all other hash types that exist for the given hash is available.

## Screenshots

branch | PR
------ | ------

## Required version of Demisto
4.5

## Does it break backward compatibility?
   - Yes
       - Further details:
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Dependencies
Mention the dependencies of the entity you changed as given from the precommit hooks in checkboxes, and tick after tested them.
- [ ] Dependency 1
- [ ] Dependency 2
- [ ] Dependency 3

## Additional changes
Describe additional changes done, for example adding a function to common server.

## Technical writer review
Mention and link to the files that require a technical writer review.
- [x] [YAML file](link)
- [ ] [CHANGELOG](link)

### CODE REVIEW

File: Playbooks/playbook-Convert_file_hash_to_corresponding_hashes_.yml [hunk 1]
```diff
@@ -0,0 +1,814 @@
+id: Convert file hash to corresponding hashes
+version: -1
+name: 'Convert file hash to corresponding hashes '
```
Comment (yaakovi) [inline Playbooks/playbook-Convert_file_hash_to_corresponding_hashes_.yml]: why do you ha e a space at the end of the name?
```suggestion
name: Convert file hash to corresponding hashes
```

File: Playbooks/playbook-Convert_file_hash_to_corresponding_hashes_.yml [hunk 2]
```diff
@@ -0,0 +1,814 @@
...
+fromversion: 4.0.0
```
Comment (yaakovi) [inline Playbooks/playbook-Convert_file_hash_to_corresponding_hashes_.yml]: shouldn't this be 4.5.0?
```suggestion
fromversion: 4.5.0
```

File: TestPlaybooks/playbook-ConvertFileHashToCorrespondingHashes-Test.yml [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    timertriggers: []
+    ignoreworker: false
+  "1":
+    id: "1"
+    taskid: 2fd612aa-4287-4576-89d2-e76416ea5602
+    type: playbook
+    task:
+      id: 2fd612aa-4287-4576-89d2-e76416ea5602
+      version: -1
+      name: 'Convert file hash to corresponding hashes '
```
Comment (yaakovi) [inline TestPlaybooks/playbook-ConvertFileHashToCorrespondingHashes-Test.yml]:
```suggestion
      name: Convert file hash to corresponding hashes
```

File: TestPlaybooks/playbook-ConvertFileHashToCorrespondingHashes-Test.yml [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+      description: |
+        The playbook allows to get all of the relevant file hash even if there is only one hash type available
+        For example if we have only the SHA256 hash the playbook will get the SHA1 and MD5 hashes
+      playbookName: 'Convert file hash to corresponding hashes '
```
Comment (yaakovi) [inline TestPlaybooks/playbook-ConvertFileHashToCorrespondingHashes-Test.yml]:
```suggestion
      playbookName: Convert file hash to corresponding hashes
```

File: TestPlaybooks/playbook-ConvertFileHashToCorrespondingHashes-Test.yml [hunk 5]
```diff
@@ -0,0 +1,124 @@
+id: Test Convert file hash to corresponding hashes
```
Comment (yaakovi) [inline TestPlaybooks/playbook-ConvertFileHashToCorrespondingHashes-Test.yml]: this should also have `fromversion: 4.5.0`

File: Playbooks/playbook-Convert_file_hash_to_corresponding_hashes_.yml [hunk 6]
```diff
@@ -0,0 +1,814 @@
+id: Convert file hash to corresponding hashes
```
Comment (yaakovi) [inline Playbooks/playbook-Convert_file_hash_to_corresponding_hashes_.yml]: why does the file name ends with underscore?

File: Tests/conf.json [hunk 7]
```diff
@@ -1853,6 +1853,11 @@
         {
             "playbookID": "Request Debugging - Test",
             "fromversion": "5.0.0"
+        },
+                {
```
Comment (yaakovi) [inline Tests/conf.json]: please fix indentation here. it's not an issue but looks bad

Comment (David-BMS) [reply Tests/conf.json]: Fixed thanks

**Comment by yaron-libman** [comment]
Reviewed the yaml file.

**Pull request merged**